.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,53 +1,53 @@
 name: Bug Report
 description: Report a bug to Hypercrx
-title: "[Bug] "
+title: '[Bug] '
 labels: [kind/bug]
 body:
-- type: input
-  attributes:
-    label: Version
-    description: |
-      Check if the issue is reproducible with the latest stable version of Hypercrx.
-    placeholder: |
-      e.g. 1.5.5
+  - type: input
```

Comment:
Can we double check the format here? Looks like we didn't change any content. We don't have changes in prettier config. This should not happen. (Or at leaset not be included in this PR)